curvefs/src/mds/topology/topology_manager.cpp

```diff
@@ -644,14 +664,27 @@ void TopologyManager::CreatePartition(const CreatePartitionRequest *request,
         }
 
         // get leader
+        PoolIdType poolId = copyset.GetPoolId();
+        CopySetIdType copysetId = copyset.GetId();
         std::string leader;
         FSStatusCode retcode = metaserverClient_->GetLeader(poolId, copysetId,
                                                     copysetMemberAddr, &leader);
         if (FSStatusCode::NOT_FOUND == retcode) {
-            LOG(WARNING) << "Get leader failed when create partition.";
+            LOG(INFO) << "Get leader failed when create partition.";
             continue;
         }
 
+        // calculate partition number of this fs
+        uint32_t pNumber = topology_->GetPartitionNumberOfFs(fsId);
+        uint64_t idStart = pNumber * option_.idNumberInPartition;
```

Comment:
root inode id is 1， so the first partition is start from 1

Reply:
> root inode id is 1， so the first partition is start from 1

This is checked and dealed in metaserver